Chef/Cookbooks/Relativity/attributes/default.rb

```diff
@@ -6,29 +6,30 @@
 default['windows']['user']['admin']['password'] = ENV['devvm_default_password']

```

Comment:
Can we add a comment at the end of the line saying this env variable is stored on the DevVM Base Image.

Reply:
Done